docs/Operators.md

```diff
@@ -7334,13 +7334,14 @@ expect(node, inputs=[x], outputs=[y], name="test_depthtospace_example")
 
 ### <a name="DequantizeLinear"></a><a name="dequantizelinear">**DequantizeLinear**</a>
 
-  The linear dequantization operator. It consumes a quantized tensor, a scale, and a zero point to compute the full precision tensor.
-  The dequantization formula is `y = (x - x_zero_point) * x_scale`. `x_scale` and `x_zero_point` must have same shape, and can be either a scalar
-  for per-tensor / per layer quantization, or a 1-D tensor for per-axis quantization.
-  `x_zero_point` and `x` must have same type. `x` and `y` must have same shape. In the case of dequantizing int32,
+  The linear dequantization operator. It consumes a quantized tensor, a scale, and a zero point to compute the full-precision tensor.
```

Comment:
I'm thinking whether it is better to create 2 new operators: QuantizeLinearBlockwise and DequantizeLinearBlockwise. The reason is it will be a burden to implement blockwise for other types while they are not used in practice.

Reply:
Blocked quantization is not limited for a single data type, as it can improve quantization accuracy for any quantized data type (at the cost of additional storage).